docs/src/explanation/troubleshooting.md

```diff
@@ -0,0 +1,26 @@
+# Troubleshooting
+
+This page provides techniques for troubleshooting common Canonical Kubernetes
+issues.
+
+## Invalid Kubeconfig
+
+### Problem
+
+The Kubernetes config file generated by the `k8s` CLI can not be used to access
```

Comment:
Please add a link here to the upstream docs that explain what a kubeconfig file is.